includes/vendor/a5hleyrich/wp-background-processing/license.txt

```diff
@@ -0,0 +1,280 @@
+        GNU GENERAL PUBLIC LICENSE
```

Comment:
So introducing background processing will require us to change the license for the WordPress plugin to any of these http://gplv3.fsf.org/wiki/index.php/Compatible_licenses ? That has some legal issues that need to be resolved then.

Reply:
Code-wise everything is fine, I'll create an issue to take a look at the licensing.